CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+**2.6.0** (5 Aug 2015)
+- add `time_event` method
+- update snippet to load mixpanel correctly from cdn when ﬁle is opened locally
```

Comment:
Fair points. Should just be a matter of writing the appropriate function and adding it to `VALIDATIONS` in the main block of `pre-commit`; I won't have time to work on it anytime soon but would be happy to thumbs-up your PR!